### ISSUE TITLE
[F-R] Space between the text and the borders settable

### ISSUE THREAD

**Issue by Palloxin** [shosetsuorg/shosetsu#192]
**Is your feature request related to a problem? Please describe.**
Make an option that would allow the variation of space between the borders and the text in the reader.

![IMG_20220427_120345](https://user-images.giuthubusercontent.com/75091899/165494338-75d89a9b-0772-44f5-86ad-5aa50601a9e3.jpg)
as yo can see the space is bare minimum, so i would like to make it variable since it is uncomfortable to read text sticked to the border.

**Comment by InitialDet** [comment]
Yeah for me thats the biggest issue with the app rn. The text is too close the border making it very uncomfortable go read

**Comment by TechnoJo4** [comment]
Use Custom CSS to add margins — you can ask in the discord server if you want any help with that.

**Comment by tak2hu** [comment]
didn't know custom css was possible, thanks @TechnoJo4

For anyone who wants have  margins like mine 

![screen](https://user-images.githubusercontent.com/73519753/187591234-03c0ccf6-e7f6-4b0a-a7ef-bf9482a4816e.png)


Go to `more` > `Settings` > `Reader` > `Html CSS` > `Open in`

Type (or copy) the following:
```
* {
margin-left: 2%;
margin-right: 2%;
text-align: justify;
text-justify: auto; 
}
```
Then tap the big save button on the middle, exit and done.
